src/System.Management.Automation/engine/NativeCommandProcessor.cs

```diff
@@ -185,6 +185,17 @@ internal NativeCommandProcessor(ApplicationInfo applicationInfo, ExecutionContex
 
             //Create input writer for providing input to the process.
             _inputWriter = new ProcessInputWriter(Command);
+
+            try
+            {
+                Host.BufferCell[,] bufferContents = this.Command.Context.EngineHostInterface.UI.RawUI.GetBufferContents(
+                    new Host.Rectangle(_startPosition, _startPosition));
+                _scrapeHostOutput = true;
+            }
+            catch (NotImplementedException)
+            {
+                _scrapeHostOutput = false;
+            }
```

Comment:
from line 444 below, it looks like a `HostException` also indicates `The host doesn't support scraping via its RawUI interface` ...